en/06_Project_Governance/06_Supported_Modules.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Supported modules
+---
+
+# Commercially supported modules for Silverstripe CMS 5
```

Comment:
```suggestion
# Commercially supported modules
```
No need to say which version it's for - the docs site already does that. This is just another heading we'd have to update next major for no benefit.